formfactory/templates/formfactory/form_detail.html

```diff
@@ -1,11 +1,12 @@
 {% extends "base.html" %}
+{% load formfactory_tags %}
 
 {% block content %}
-    <form class="Form" method="POST" action="." enctype="multipart/form-data">
-        {% if view.form_object.enable_csrf %}
-            {% csrf_token %}
-        {% endif %}
-        {{ form.as_p }}
-        <input type="submit" value="{{ view.form_object.submit_button_text }}" />
-    </form>
+    {% comment %}
+        We by default use the render form tag here. This means we don't need
+        duplicate js for ajax posting on base forms. Also allowing users to use
+        the existing detail form and only needing to override the inner
+        inclusion detail template.
+    {% endcomment %}
+    {% render_form object %}
```

Comment:
Opted for duplicate js in base detail. Rather than have a view level hook for ajax detail templates as this might cause issues for an end user somewhere along the line. 

Did add the ability to pass a template suffix via url however.